packages/css-framework/src/components/_phase-banner.scss

```diff
@@ -0,0 +1,25 @@
+.rn-phase-banner {
+  display: block;
+  background-color: color(primary, 100);
+  padding: spacing(2) spacing(6);
+  &--alpha {
+    display: block;
+  }
+
+  &--beta {
+    display: block;
+  }
+
+  &__text {
+    display: inline-block;
+    margin-left: spacing(1);
+    font-size: font-size(xs);
+    a {
+      color: color(neutral, black);
+    }
+  }
+
+  .rn-badge--solid {
```

Comment:
Can we vertically align the text with the badge? At the moment the "This is a new service..." text is higher than the alpha button. Ideally the text and badge are both vertically centred in the banner